packages/syft/src/syft/service/worker/worker_pool.py

```diff
@@ -78,12 +109,77 @@ class WorkerPool(SyftObject):
     __version__ = SYFT_OBJECT_VERSION_1
 
     __attr_unique__ = ["name"]
-    __attr_searchable__ = ["name", "syft_worker_image_id"]
+    __attr_searchable__ = ["name", "image_id"]
+    __repr_attrs__ = [
+        "name",
+        "image_id",
+        "max_count",
+        "workers",
+        "created_at",
+    ]
 
     name: str
-    syft_worker_image_id: UID
+    image_id: UID
+    image: SyftWorkerImage
```

Comment:
`image: SyftWorkerImage` already contains `image_id` here (`image_id` here is the same as `image.id` right) so `image_id: UID` is redundant here.

Reply:
That's true, thanks @kiendang. Updated the code